docs/deployment/helm.md

```diff
@@ -44,26 +44,39 @@ repository and install from there.
 ```bash
 git clone https://github.com/kubernetes-sigs/node-feature-discovery-operator/
 cd node-feature-discovery-operator/deployment/helm
-export NFD_O_NS=nfd-operator
+export NFD_O_NS=node-feature-discovery-operator
 helm install nfd-operator ./nfd-operator/ --namespace $NFD_O_NS --create-namespace
 ```
 
 See the [configuration](#configuration) section below for instructions how to
 alter the deployment parameters.
 
+By default the operator will watch `NodeFeatureDiscovery` objects
+only in the namespace where the operator is deployed in. This is
+specified by the `WATCH_NAMESPACE` env variable in the operator
+deployment manifest. If unset the operator will watch ALL
```

Comment:
I don't see the operator behaving this way 🧐 Should it?

Reply:
ah ups, that's old school behav